Library/Formula/generate-json-schema.rb

```diff
@@ -15,10 +15,24 @@ class GenerateJsonSchema < Formula
 
   depends_on "node"
 
+  resource "npm-type-of-is" do
+    url "https://registry.npmjs.org/type-of-is/-/type-of-is-3.4.0.tgz"
+    sha256 "5e09fc1d5968ef748ecac3d048244cc2a28ef4e6b503342662d0734498d166b8"
+  end
+
   def install
-    ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"
+    npm = "#{Formula["node"].opt_libexec}/npm/bin/npm"
+    # let us manage the dependencies
+    inreplace "package.json",
+      /"dependencies": \{[^}]+\},?/, ""
```

Comment:
I was able to shorten this by using `--only=production` but there must be a way to remove this `inreplace` and use an option.